client/app/dedicatedCloud/service-pack/upgrade/steps/validation/validation.html

```diff
@@ -0,0 +1,20 @@
+<p data-translate="dedicatedCloud_servicePack_summary_stepper_go_management_interface"></p>
+
+<a
+    class="oui-button oui-button_primary"
+    data-ng-href="{{:: $ctrl.currentService.certifiedInterfaceUrl }}"
+    rel="noopener"
+    target="_blank"
+    data-translate="dedicatedCloud_servicePack_summary_stepper_go_management_interface_action"></a>
```

Comment:
What do you think about adding a `<span class="sr-only"></span>` that tell to the user that this link will open a new window?